projects/plugins/jetpack/changelog/fix-search-esc-binding

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Instant Search: Prevent escape key from stopping page reload
```

Comment:
```suggestion
Instant Search: ensure Escape key always closes search modal
```

Reply:
I like it. Committing this, and then it should be good to merge!